content/en/docs/tasks/security/authz-http/index.md

```diff
@@ -94,98 +119,170 @@ There may be some delays due to caching and other propagation overhead.
 Remove the following configuration before you proceed to the next task:
 
 {{< text bash >}}
-$ kubectl delete -f @samples/bookinfo/platform/kube/rbac/namespace-policy.yaml@
+$ kubectl delete authorizationpolicy.security.istio.io/bookinfo-viewer
 {{< /text >}}
 
-## Enforcing Service-level access control
+## Enforcing workload-level access control
 
-This task shows you how to set up service-level access control using Istio authorization. Before you start, please make sure that:
+This task shows you how to set up workload-level access control using Istio authorization.
```

Comment:
The introduction needs some additional explanation. Why are you starting with a denying all policy? What is the **best** flow to enforce access control at this level?
Generally, the task does not let readers know why they should use each level of access control. What are the goals at each level? When do I apply restrictions on each level?

Reply:
+1. I think instead of explaining the 3 different scopes, it is better to show how the authz policies match (to workload) and combine.

Reply:
+2. We can start with some introduction explaining the steps. So readers will get a picture of what this task demonstrates before delving into each step.

Reply:
I added a high-level summary for the big picture of the task.

@diemtvu For now I'm converting the old task that was focusing on the scope, I think we can rewrite to make this more useful by also talking explaining more about the selector part, but let's do it in a follow-up PR to avoid introducing too much new stuff in a single PR.

Reply:
I think it's ok to rewrite the story in one PR.

Reply:
In terms of rewrite, I actually want to use sleep/httpbin in multiple namespaces demonstrate more features of the API, but for now I do not have enough time for the complete new task, we still need to update the migration/troubleshooting/etc. docs.

@liminw thoughts?

Reply:
The new application I'm thinking is `httpbin` and `sleep` which is currently used in many places in both networking tasks and the authentication task. The `httpbin` is much smaller and gives better deterministic response which helps a lot in automating the tests, it's not maintained by Istio and is a public tool used widely in many other projects.

I will not change this in this PR but I think it's a better replacement for the Bookinfo in the future.